src/test-fn/kotlin/com/anaplan/engineering/vdmgradleplugin/UpToDateTest.kt

```diff
@@ -741,6 +741,16 @@ class generateLaunchGenForAllTestUpToDateTest {
         ), expectBuildFailure = true)
     }
 
+    @Test
+    fun test_noFiles() {
```

Comment:
This is not technically an "up-to-date" test, but it was easier to write it using the utility functions from this class.